rpc/metrics.go

```diff
@@ -23,9 +23,9 @@ import (
 )
 
 var (
-	rpcRequestGauge        = metrics.NewRegisteredGauge("rpc/requests", nil)
-	successfulRequestGauge = metrics.NewRegisteredGauge("rpc/success", nil)
-	failedReqeustGauge     = metrics.NewRegisteredGauge("rpc/failure", nil)
+	rpcRequestGauge        = metrics.GetOrRegisterGauge("rpc/requests", nil)
```

Comment:
moreover, if go-eth/rpc package also imported, better to ensure go-eth/rpc initialized at first.

So, you can explicitly import eth/rpc package in this file to ensure metrics are initialized correctly.